webpack.config.js

```diff
@@ -36,7 +37,9 @@ const base = {
             })
         ]
     },
-    plugins: []
+    plugins: [
+      new NormalModuleReplacementPlugin(/^mqtt$/, "mqtt/dist/mqtt.js")
```

Comment:
For some reason I couldn't get webpack to compile the `mqtt` dependency, so I had to manually refer it to the compiled source.

see https://github.com/mqttjs/MQTT.js/issues/720